config/initializers/devise.rb

```diff
@@ -0,0 +1,290 @@
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'e3c2505a8e1b8e53487692b2f5de3dbbf5ae51d167aa2d6ec55f9a93635d1846eb3bcbbbb7373392ebcef0511e14494eeca05681f6af4300fd75313eaafa238c'
+  
+  # ==> Controller configuration
+  # Configure the parent class to the devise controllers.
+  # config.parent_controller = 'DeviseController'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
Let's change this to something more meaningful 😛 What do you think about fetching this from an ENV variable using Rails secrets? [Here](https://brandonhilkert.com/blog/using-rails-4-dot-1-secrets-for-configuration/) is some info about that!